src/components/LibraryItem.tsx

```diff
@@ -184,9 +185,6 @@ export class LibraryItem extends React.Component<LibraryItemProps, LibraryItemSt
     }
 
     onLibraryItemClicked() {
-
-        // Toggle expansion state.
-        let currentlyExpanded = this.state.expanded;
-        this.setState({ expanded: !currentlyExpanded });
+        LibraryEntryPoint.dispatchEvent("libraryItemClick", this);
```

Comment:
We'll leave that to the `index.html` or any other clients who are using `librarie.js`